packages/amplify-graphql-schema-generator/src/ts-schema-generator/helpers.ts

```diff
@@ -225,25 +232,28 @@ export const createSchema = (schema: Schema, config?: DataSourceGenerateConfig):
       return createModel(model, engine);
     });
 
-  const combinedEnums = nullEnums.concat(notNullEnums).flat() // making 1 D array
+  const combinedEnums = nullEnumFields.concat(RequiredEnumFields).flat(); // making 1 D array
 
-  if (engine === 'Postgres') { // for postgresql
+  /** 
+  for postgresql since enums are declared globally to void declaring duplicate enums we
+  declare find unique enums and then pass it to the schema  
```

Comment:
I can't tell what you mean with this explanation. Can you reword to clarify? (I mean to say, based on our discussion and reading through the code, I know what you're trying to do--it's the comment wording that is unclear.)

Also, nit: reserve the double-asterisk block comment structure (`/** ... */` for docComments (such as we put at the beginning of an exported symbol). Other comments should be delimited with a double-slash (`//`)

Reply:
Sure I will re word this and and remove the double-asterisk block structure

Reply:
Added appropriate comment structure and comments to explain in brief the use case